templates/eks/amazon-eks-nodepool-cf.yaml

```diff
@@ -274,7 +274,7 @@ Resources:
         !Ref Subnets
       Tags:
       - Key: Name
-        Value: !Sub "${ClusterName}-${NodeGroupName}"
+        Value: !Sub "${ClusterName}-${NodeGroupName}-Node"
```

Comment:
Is this going to be backward compatible?

Reply:
It's not in terms of, that asg's made with previous CF template will have different tags, however it doesn't seem to effect anything, at least nodes are joining with this new tag as well to EKS master, regardless of k8s version.